### ISSUE TITLE
models/ctr跑完auc只有0.52左右

### ISSUE THREAD

**Issue by buptwds** [PaddlePaddle/models#486]
代码用的models/ctr，数据集用的是代码中推荐的：Kaggle上CTR任务的数据集，使用avazu_data_processer.py处理后保留500w左右的数据用于train，infer数据集使用默认的100条左右，auc使用sklearn.metrics.roc_auc_score计算，但是auc只有0.52左右，是哪里出错了吗，还是只能达到这个效果？

**Comment by QiJune** [comment]
请详细描述问题，明确的标题与详细的说明有助于解答

**Comment by QiJune** [comment]
请 @Superjom 帮忙确认一下

**Comment by Superjomn** [comment]
1. 请确认 training cost 可以下降到很低，否则是数据问题
2. 如果 training AUC比较高，但 test AUC 比较低，确认是否是过拟合问题

请先确认数据本身，任务类型没有问题。

**Comment by buptwds** [comment]
数据使用的是avazu_data_processer.py默认产生的数据

**Comment by Superjomn** [comment]
请补充任务更多信息，比如训练/测试数据如何生成，训练和测试等命令，方便定位问题，尽量详细些。

没法从一两句话猜出你的问题。

**Comment by buptwds** [comment]
num_pass使用的是默认的10，infer的模型使用的是ctr_models-pass-9-batch-0-test-xxxx.tar.gz

**Comment by Superjomn** [comment]
比如训练/测试数据如何生成，训练和测试等命令

最好用一段比较详细的描述。

**Comment by buptwds** [comment]
了解models/ctr例子的同学应该知道，下面有详细的数据下载路径，数据生成方式，train、infer的命令，可参考https://github.com/PaddlePaddle/models/tree/develop/ctr

**Comment by buptwds** [comment]
https://github.com/PaddlePaddle/models/tree/develop/ctr
里面的README是paddle的相关同学写的，我想问的是你们自己按照这里的步骤，跑出来的结果是多少，因为我按照里面的步骤跑出来的结果不好，所以才有疑问的

**Comment by Superjomn** [comment]
正在下载数据重现下，有结果更新到这里

**Comment by buptwds** [comment]
请问这个问题有进展吗

**Comment by Superjomn** [comment]
fixed.

**Comment by shuDaoNan9** [comment]
我也遇到了，我是直接用的“https://github.com/PaddlePaddle/models/tree/develop/PaddleRec/ctr/deepfm”的代码，按照上面readme步骤运行的，啥都没改，跑出来测试集指标只有“loss: [0.84420469] auc_val: [0.51261696]”，0.5跟readme说的auc是0.8差距较大；我自己用TensorFlow写的auc算的0.76
